srtcore/core.cpp

```diff
@@ -8952,6 +8983,20 @@ int CUDT::packLostData(CPacket& w_packet, steady_clock::time_point& w_origintime
             continue;
         }
 
+        if (m_bPeerNakReport && m_iOPT_RexmitAlgo != 0)
+        {
+            steady_clock::time_point tsLastRexmit;
+            m_pSndBuffer->getPacketTime(offset, tsLastRexmit);
+            if (!is_zero(tsLastRexmit) && tsLastRexmit >= time_nak)
```

Comment:
Testing for `is_zero` is superfluous here, as zero time will still be "in the past" towards `time_nak`.